lib/backup/encryptor/open_ssl.rb

```diff
@@ -62,8 +62,13 @@ def options
         opts = ['aes-256-cbc']
         opts << '-base64' if @base64
         opts << '-salt'   if @salt
-        opts << ( @password_file.to_s.empty? ?
-                "-k '#{@password}'" : "-pass file:#{@password_file}" )
+
+        if @password_file.to_s.empty?
+          opts << "-k #{Shellwords.escape(@password.to_s)}"
```

Comment:
Ruby 1.9.3 is no longer supported https://www.ruby-lang.org/en/news/2015/02/23/support-for-ruby-1-9-3-has-ended/

So we don't support it either anymore. But I see I should probably take it out of the travis.yml file.. So... Done :) https://github.com/backup/backup/commit/7628b757998592c67c0b7b920f2fe0bdbbab9509